src/v/coproc/reconciliation_backend.cc

```diff
@@ -88,7 +130,9 @@ reconciliation_backend::process_events_for_ntp(
         vlog(coproclog.trace, "executing ntp: {} op: {}", d.ntp, d);
         auto err = co_await process_update(d);
         vlog(coproclog.info, "partition operation {} result {}", d, err);
-        if (err == errc::partition_not_exists) {
+        if (
```

Comment:
There's a bug with this method. In the event there's a retry it continues processing the next event anyway. Out of order processing will occur in this scenario